### DISCUSSION TITLE
Donation link

### DISCUSSION THREAD

**Discussion by primeapple** [coddingtonbear/inthe.am#415] [General]
Hi,

I would like to support your work. If I would a taskd server I would have to pay monthly money for the server.
Since I use your service, you pay this money instead.

Please add a possibility for donations :)

**Comment by coddingtonbear** [comment]
As much as I appreciate the sentiment, I'm extremely hesitant about accepting donations because, even if it's not directly asked for, accepting money might encourage folks into having expectations of support that I really can't reasonably provide.  I'm glad you're finding it useful, though!

**Reply by primeapple** [reply]
Thanks for the answer. It's fine then :)
I'll try to help you with contributing. But this is harder then just sending money over :wink: 

But maybe leave this discussion open, maybe there are many more folks that would like to send money, then you could think about it again!
